### ISSUE TITLE
codological unit vs. textual units

### ISSUE THREAD

**Issue by eroux** [BuddhistDigitalResourceCenter/owl-schema#54]
Some texts are grouped in different ways in different canons. A typical example could be `D7` which can be divided into two different texts (can be called `D7-1` and `D7-2`, corresponding to `N7` and `N8`, to be checked). In terms of vocabulary, the gsung rten database distinguishes *codicological units* (here `D7`) and *textual units* (here `D7-1` and `D7-2`). I'm don't have any idea yet on what would be a good way to encode all that in the outlines, but I just thought it would be worth writing that down...

**Comment by xristy** [comment]
I understand that codicology is the study of codices or manuscript books written on parchment as physical objects. So I'm not entirely sure about distinguishing the two sorts of units. I guess what is going on might be that some characteristic of codices might lead to otherwise single conceptual entities being split across two or more physical carriers. Is this what is going on? 

If so perhaps some of the examples in Kangyur where there are two or more volumes for a single work/text, e.g., the 8,000 verse perfection of wisdom sutra which is 12 volumes (14 - 25) of the sde dge, W22084, and 16 volumes (57 - 72) of co ne, W1PD96685. In the conceptual work, `bdr:T2423` (which has not been turned into a `:Work` yet), there should be _canonical_ part `:Work`s such as the 8,000 verse sutra that could be referred to by the digital surrogates of the physical carrier components of the _conceptual work_.

It seems to me that what is being considered is a general situation with compilations and is independent of whether a compilation is in manuscript or not. 

The situation seems to be one in which one compilation, `D`, has a single  _text_, perhaps labeled `d7` which appears in a second compilation, `N`, as _the concatenation_ of two _texts_, `n7` and `n8`.

Having thought for a spell and consulted the example ideas in the ontology narrative, it seems to me that we likely need a bit of information to indicate the ordering of the _physical_ components of a _conceptual_ work. We have the `:workPartOf` but we're relying on volume numbering and page numbering to supply ordering. Maybe that's enough?

Otherwise maybe we could have something like:

     d7 :workIsSimilarTo [ a rdf:Seq ; rdf:_1 n7 ; rdf:_2 n8 ] .

using the already defined `:workIsSimilarTo` property. Then to get the inverse would be like:

      n7 :workIsSimilarTo [ a :WorkCombination ; :workPart 1 ; :workPartOf d7 ] 
      n8 :workIsSimilarTo [ a :WorkCombination ; :workPart 2 ; :workPartOf d7 ]

which would introduce `:WorkCombination` and `:workPart`

**Comment by eroux** [comment]
Well, I need to inspect it more deeply, but what I understood is:
- a codological unit is a unit according to the codex one is looking at, one unit in the karchak for instance
- a textual unit is either a codologial unit or a part of it, sometimes they don't appear in the karchak for instance

so we can already encode it with `:workPartOf`, but I think it would be useful to retain the distinction between the two... I'm really not sure how to do that. But these issues I've opened don't need to be resolved right away, especially since it won't really affect migration... It's more a way to remember it

**Comment by xristy** [comment]
I appreciate keeping the issues as something to remind us of; however, the case of texts extending across multiple volumes is one that isn't completely modeled if there's a need to show how parts of the sde dge correspond to the co ne and also we do need a conceptual Kangyur structure.

**Comment by eroux** [comment]
I agree we should model that yes, maybe we can start a google doc or something like that?

**Comment by eroux** [comment]
To give an example of a possible solution (which may not be semantic enough?), we could have a convention that a sub-codological unit is indicated with `-01`, `-02` (ex: `7-01`), etc. this would make the data homogeneous. Currently different catalogues use different ways, for instance Hackett would use `(a)`, `(b)`, etc. others use `(1)`, `(2)`, etc. others `-1`, `-2`, etc. This would give roughly:

```ttl
bdr:W123 :workKaTenSiglaD "7-01" .
```

Another solution would be to get more descriptive and record something like:

```ttl
bdr:W123 :hasRef [ a :CatalogueRef ;
                                :inCatalogue bdr:DergueCatalogue ;
                                :codologicalUnit 7^^xsd:int ;
                                :subUnit 1^^xsd:int ;
                              ]
```

**Comment by eroux** [comment]
In fact I may have the vocabulary completely wrong here (codological unit, etc.) but the question of subindexes remain...
